daemon/config/config_linux.go

```diff
@@ -257,7 +257,9 @@ func setPlatformDefaults(cfg *Config) error {
 			//
 			// An error is still produced by [Config.ValidatePlatformConfig] if
 			// userland-proxy is enabled in the configuration.
-			log.G(context.TODO()).WithError(err).Warn("failed to lookup default userland-proxy binary")
+			//
+			// We log this at "debug" level, as this code is executed
```

Comment:
... when running "--version"

(Also, "Log a warning" -> "Log" in the comment above?)

Reply:
Doh! Looks like I got distracted and went writing the FIXME first 🙈. Let me fix

Reply:
Fixed; thanks for catching!